src/pages/queryBuilder/QueryBuilder.jsx

```diff
@@ -41,9 +41,8 @@ export default function QueryBuilder() {
    */
   async function onQuickSubmit() {
     pageStatus.setLoading('Fetching answer, this may take a while');
-    const ara_url = ARAs[ara];
     const prunedQueryGraph = queryGraphUtils.prune(queryBuilder.query_graph);
-    const response = await API.ara.getAnswer(ara_url, { message: { query_graph: prunedQueryGraph } });
+    const response = await API.ara.getQuickAnswer(ara, { message: { query_graph: prunedQueryGraph } });
```

Comment:
Double checking that this should now be getQuickAnswer and not getAnswer?

Reply:
Yeah, this one gets sent directly to the ara without getting saved in robokache. Line 73 sends the question id instead of the message because it's interacting with robokache.